wp_api/src/login.rs

```diff
@@ -123,12 +125,20 @@ pub struct KnownApplicationPasswordBlockingPlugin {
 
 impl KnownApplicationPasswordBlockingPlugin {
     fn all() -> Vec<Self> {
-        vec![Self {
-            name: "Wordfence".to_string(),
-            namespace: "wordfence/v1".to_string(),
-            // TODO: Ensure this is correct with the WordFence folks
-            support_url: "https://www.wordfence.com/support/".to_string(),
-        }]
+        vec![
+            Self {
+                name: "Wordfence".to_string(),
+                namespace: "wordfence/v1".to_string(),
+                // TODO: Ensure this is correct with the WordFence folks
+                support_url: "https://www.wordfence.com/support/".to_string(),
+            },
+            Self {
+                name: "Hostinger Tools".to_string(),
+                namespace: "hostinger-tools-plugin/v1".to_string(),
+                // TODO: Ensure this is correct with the Hostinger folks
+                support_url: "https://wordpress.org/support/plugin/hostinger/".to_string(),
```

Comment:
An alternative way is to hard-code the WordPress.org plugin directory slug (which is unlikely to change) instead. We can get the support URL from the plugin directory, which should always be correct.

```
curl -s 'https://api.wordpress.org/plugins/info/1.2/?action=plugin_information&slug=hostinger' | jq -r '.support_url'
```

Reply:
We could do that, but it's another `await` which I'd be nervous about.

For now, this URL won't change (IINM plugin slugs are immutable in the directory) but I'm pretty sure we'll end up sending them to something like `https://support.hostinger.com/$SOME_PATH`